src/model/Expression.js

```diff
@@ -18,6 +18,9 @@ class Expression {
     return null;
   }
 
+  getVariables(){
+    return null;
+  }
```

Comment:
Do not return `null`, if you can return a sane type-safe default – `[]` in this case.